views/components/AndroidSignUp.js

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from 'react';
+
+export default class AndroidSignUp extends Component {
+  render() {
+    return (
+      <div className="container-splash-lead">
+        <div className="container-splash">
+          <div className="splash-top">
+            <img
+              src="/images/app-page/LaunchPage-Logo-Mobile.png"
+              className="splash-logo"
+            />
+            <div className="splash-header">Hear it first.</div>
+            <div className="splash-subtitle">
+              Our Android app is in the works—sign up to be the{' '}
+              <strong>first</strong> to find out when it launches.
+            </div>
+          </div>
+          <div className="splash-signup-form">
+            <form action="/android-signup" method="post">
+              <div className="splash-form-field">
+                <input
+                  className="form-control"
+                  value=""
+                  type="text"
+                  name="FNAME"
+                  id="mce-FNAME"
+                />
+                <label htmlFor="mce-FNAME">First Name</label>
```

Comment:
And since this and the email field are going to be required, I'd recommend putting a `*` into the label too in order to indicate that - ie: `First Name *`.